tls/extensions/s2n_server_supported_versions.h

```diff
@@ -21,4 +21,4 @@
 
 int s2n_extensions_server_supported_versions_recv(struct s2n_connection *conn, struct s2n_stuffer *extension);
 int s2n_extensions_server_supported_versions_send(struct s2n_connection *conn, struct s2n_stuffer *out);
-int s2n_extensions_server_supported_versions_size(struct s2n_connection *conn);
```

Comment:
Is this worth changing? Conn isn't used, but the _size() function does match the signature of the other extensions.

Reply:
i don't see a reason to keep conn in there

Reply:
i added `conn` in #1672 to match other signatures, and allows the interface to be unified later

Reply:
I'm with Josh on this one. We've established basically an interface for extensions, and will likely treat is as such later with a lookup table of function pointers. When we do that, we'll just have to re-add connection. At that point the function will have gone through an uncomfortable number of flip-flops: original no-conn -> add conn to match other extensions -> remove conn because it's unnecessary -> add conn back so we can use the right function pointer.

Reply:
But we probably do need a comment explaining why it has connection as a argument, so nobody else decides to clean it up.